include/xsparse/levels/compressed.hpp

```diff
@@ -18,20 +19,32 @@ namespace xsparse
                   class IK,
                   class PK,
                   class ContainerTraits
-                  = util::container_traits<std::vector, std::unordered_set, std::unordered_map>>
+                  = util::container_traits<std::vector, std::unordered_set, std::unordered_map>,
+                  class LevelProperties = level_properties<true, true, true, false, true>>
         class compressed;
 
-        template <class... LowerLevels, class IK, class PK, class ContainerTraits>
-        class compressed<std::tuple<LowerLevels...>, IK, PK, ContainerTraits>
+        template <class... LowerLevels,
+                  class IK,
+                  class PK,
+                  class ContainerTraits,
+                  class LevelProperties>
+        class compressed<std::tuple<LowerLevels...>, IK, PK, ContainerTraits, LevelProperties>
             : public level_capabilities::coordinate_position_iterate<compressed,
                                                                      std::tuple<LowerLevels...>,
                                                                      IK,
                                                                      PK,
-                                                                     ContainerTraits>
+                                                                     ContainerTraits,
+                                                                     LevelProperties>
 
         {
-            using BaseTraits = util::
-                base_traits<compressed, std::tuple<LowerLevels...>, IK, PK, ContainerTraits>;
+            static_assert(LevelProperties::is_branchless == false);
+            static_assert(LevelProperties::is_compact == true);
```

Comment:
```suggestion
            static_assert(!LevelProperties::is_branchless);
            static_assert(LevelProperties::is_compact);
```

Same pattern everywhere else.